renovate.json

```diff
@@ -0,0 +1,272 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":gitSignOff",
+    ":disableDependencyDashboard"
+  ],
+  "ignorePresets": [
+    ":dependencyDashboard"
+  ],
+  "onboarding": false,
+  "requireConfig": "optional",
+  "inheritConfig": true,
+  "platformCommit": "enabled",
+  "autodiscover": false,
+  "vulnerabilityAlerts": {
+    "enabled": false
+  },
+  "enabledManagers": [
+    "tekton",
+    "dockerfile",
+    "rpm",
+    "custom.regex",
+    "argocd",
+    "crossplane",
+    "fleet",
+    "flux",
+    "helm-requirements",
+    "helm-values",
+    "helmfile",
+    "helmsman",
+    "helmv3",
+    "jsonnet-bundler",
+    "kubernetes",
+    "kustomize",
+    "asdf",
+    "fvm",
+    "git-submodules",
+    "hermit",
+    "homebrew",
+    "nix",
+    "osgi",
+    "pre-commit",
+    "vendir",
+    "terraform",
+    "terraform-version",
+    "terragrunt",
+    "terragrunt-version",
+    "tflint-plugin",
+    "pep621",
+    "pip-compile",
+    "pip_requirements",
+    "pip_setup",
```

Comment:
is this a default list of managers? I don't think we need pip, ATM.

Reply:
Good point. Since this is the default config, I think it might be best to leave it as is, and remove the pip manager if it creates any noise. Just to make updating to newer configs easier.